runtime-modules/working-group/src/lib.rs

```diff
@@ -120,6 +120,9 @@ pub trait Trait<I: Instance = DefaultInstance>:
 
     /// Minimum stake required for an opening
     type MinimumStakeForOpening: Get<Self::Balance>;
+
+    /// Stake needed to create an opening
+    type OpeningStake: Get<Self::Balance>;
```

Comment:
Consider renaming to LeaderOpeningStake, OpeningStakeForLeader, or similar because the council doesn't need to stake anything.